src/app/store.ts

```diff
@@ -1,11 +1,18 @@
-import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
+import {
+  configureStore,
+  ThunkAction,
+  Action,
+  combineSlices,
```

Comment:
There is a typo in the function name `combineSlices`. It should be `combineReducers` to correctly combine the reducers.